src/data/agile-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const agileQuiz = [
+{
+  Question:"In Agile, what are epics?",
+  Answer:"epics are a way to group pieces of work together to represent a feature.",
+  Distractor1:"epics are another term for scrum master",
+  Distractor2:"epics do not exist in agile",
+  Distractor3:"epics are another term for standup meetings",
+  Explanation:" Stories have a goal of defining a bite-sized piece of work, epics are a way to group those pieces of work together to represent a feature.",
+  Link:"https://www.freecodecamp.org/news/what-is-agile-and-how-youcan-become-an-epic-storyteller/#sprints"
+},
+
```

Comment:
You have this question in here twice.
Please delete this entry

Reply:
@erickleon88 

I went ahead and removed this duplicate question.
But make sure to be extra careful to go through all of the review comments and make the changes requested 👍